test/TestProjects/MgmtTypeSpec/samples/Generated/Samples/Sample_FooCollection.cs

```diff
@@ -24,7 +24,7 @@ public partial class Sample_FooCollection
         public async Task CreateOrUpdate_CreateAFoo()
         {
             // Generated from example definition: 2024-05-01/Foos_CreateOrUpdate.json
-            // this example is just showing the usage of "Foo_CreateOrUpdate" operation, for the dependent resources, they will have to be created separately.
```

Comment:
It should be `Foos`, not `Foo` which is a regression when upgrading to tsp `0.61.2`: https://github.com/Azure/autorest.csharp/pull/5110/files#diff-22fec97a025df3dec39b67b814688ef812fb9f7dfd2f85b3cc48d2501823415b
![image](https://github.com/user-attachments/assets/cd7cae6d-6ac6-4b7d-880d-a7b010911dc6)